alphalens/tears.py

```diff
@@ -303,7 +303,7 @@ def next_cell(self):
                                                                                     prices,
                                                                                     periods_before=before,
                                                                                     periods_after=after,
-                                                                                    demeaned=long_short)
+                                                                                    demeaned=True)
```

Comment:
That is actually a bug fix.

As we are calling the function for each group separately (as we are in the "Group Specific Breakdown" code), we want returns demeaning to occur on the group level. Few lines above the code does the same when it forces demeaning on perf.mean_return_by_quantile.